<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassengerHomePage</class>
 <widget class="QWidget" name="PassengerHomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="profilButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Profil</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::UserAvailable"/>
   </property>
  </widget>
  <widget class="QPushButton" name="deconnecterButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Déconnexion</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemShutdown"/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QComboBox" name="comboBoxVilleDepart">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>82</width>
         <height>28</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Départ</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxVilleArrivee">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>82</width>
         <height>28</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Arrivé</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxNombrePlaces">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>82</width>
         <height>28</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Places</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="appliquerButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>70</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditFind"/>
       </property>
      </widget>
      <widget class="QPushButton" name="dateButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>81</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="gridVisible">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QCalendarWidget::SelectionMode::SingleSelection</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::VerticalHeaderFormat::ISOWeekNumbers</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetResults">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>381</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
